iOS/README.rst

```diff
@@ -131,10 +132,21 @@ In this invocation:
   Python, you can compile a python interpreter and then use that interpreter to
   run Python code. However, the binaries produced for iOS won't run on macOS, so
   you need to provide an external Python interpreter. This interpreter must be
-  the version as the Python that is being compiled.
-
-In practice, you will likely also need to specify the paths to iOS builds of the
-binary libraries that CPython depends on (XZ, BZip2, LibFFI and OpenSSL).
+  the *exact* same version as the Python that is being compiled.
+
+For a full CPython build, you also need to specify the paths to iOS builds of
+the binary libraries that CPython depends on (XZ, BZip2, LibFFI and OpenSSL).
+This can be done by defining the ``LIBLZMA_CFLAGS``, ``LIBLZMA_LIBS``,
+``BZIP2_CFLAGS``, ``BZIP2_LIBS``, ``LIBFFI_CFLAGS``, and ``LIBFFI_LIBS``
```

Comment:
I guess SQLite isn't in this list because it's included with iOS?

Reply:
Correct. SQlite3 is used by CoreData, which is the built-in data modelling and persistence layer on macOS and iOS. The test_sqlite3 tests all pass without any additional configuration.